src/angular/loader/loader.service.ts

```diff
@@ -20,15 +22,12 @@ export class LoaderService {
         }
     }
 
-    public activate(name: string = 'general') {
-        if (this.registeredLoaders[name]) {
-            this.registeredLoaders[name].activate();
-        }
+    public activate(name: string) {
+        this.registeredLoaders[name] ? this.registeredLoaders[name].activate() : this.registeredLoaders[this.mainLoaderName].activate(); 
```

Comment:
This isn't true... roll back.
Just give a default value for the name parameter: `name: string = this.mainLoader`.
Or: `name = (name !== undefined) ? name : this.mainLoader;`
[And keep the old if...]